docs/build/web.md

```diff
@@ -11,9 +11,11 @@ redirect_from: /docs/how-to/build/web
 
 There are 2 steps to build ONNX Runtime Web:
 
-- build ONNX Runtime for WebAssembly
-  - or skip and download a pre-built artifacts
-- build onnxruntime-web (NPM package)
+- Obtaining ONNX Runtime WebAssembly artifacts - can be done by - 
```

Comment:
I like this clarification. I think it would be even clearer if we put the onnxruntime-web build step first, both here and in th body of the docs. That way the easiest scenario is catered for first.

I think we also update the section title "Build ONNX Runtime for web assembly" to "Build ONNX Runtime web assembly artifacts"

Reply:
Thanks for the feedback :). I agreed that the new title is clearer - I updated the PR with your suggestion.
Can you please elaborate a bit more on your suggestion for the steps sequence ? I think you wanted to see the build artifacts as the first step - but I think it's already first in line.

Reply:
Sure. RIght now we have the order as:
1. Obtain or build the WASM artifacts
2. Build onnxruntime-web

I'm suggesting we reverse that order, as the simplest use case is to build the package with the pre-built WASM artifacts ie

1. Build onnxruntime-web
2. Obtain or build WASM artifacts

Reply:
I agree that the overall process still needs some more clarification. Changing the steps will not work since Building the onnxruntime web package needs the WASM artifacts. We were talking internally in the team and we are working on a visualization for the process. this is the current image we have - 
![ORTWeb_build_process](https://user-images.githubusercontent.com/94086448/184235822-a3fba2c4-419f-43e0-bde6-2de03c45d878.jpg)

Will this image help clarify and address your concern ? please share your feedback